src/smtml/mappings.ml

```diff
@@ -903,13 +903,24 @@ module Make (M_with_make : M_with_make) : S_with_fresh = struct
           Stack.push ctx s.ctx;
           M.Solver.add s.solver ~ctx exprs
 
+      module Tmp_log = Tmp_log_path
```

Comment:
This is used only once, you can probably remove this alias to avoid useless cognitive burden